### ISSUE TITLE
Feature: Display number of captured events in the Captures Collected window

### ISSUE THREAD

**Issue by Pursche** [baldurk/renderdoc#1723]
## Description

I have encountered the _very_ minor inconvenience of capturing the Unreal Engine editor using RenderDoc. While running through the editor, the editor tries to render in a fixed framerate regardless of if the game itself manages to render that fast. It does this by rendering the game to a texture as fast as possible, and then displaying that texture every frame.

So, when I take captures from the UE4 editor, sometimes it doesn't actually capture the _game_ being rendered, just the editor. This is easily solvable by taking more captures until I manage to hit the scene being rendered. However, the correct captures are pretty large with around 20 000 events, so loading the captures is not instant.

If the "Captures collected:" window, (which displays the outputted texture, the name of the program, the rendering API, the frame number and a timestamp), could also display the number of events in the capture I could easily see that the capture I just took only has around 1000 events, so I could take a new capture immediately instead of having to open it before noticing.

Once again, this is a _very_ minor feature request for an almost perfect tool, thank you for creating RenderDoc.

## Environment

* RenderDoc version: v1.6
* Operating System: Windows
* Graphics API: DX12

**Comment by baldurk** [comment]
Displaying the number of events is not feasible because you'd need to serialise most of the. What about the file size, would that sufficiently disambiguate the captures for you? That could be done cheaply enough.

Assuming you're using the unreal plugin though, I think the best fix for this is on the unreal side to only perform captures around 'real' work and not the editor's tick. i.e. force a full render when capturing.

**Comment by Pursche** [comment]
Yeah I think the file size would be sufficient.

I did use the unreal plugin, but it absolutely kills editor performance down to a point where I can't even do basic tasks so I ended up disabling it and injecting manually, this has some performance cost but not nearly as much as the plugin. I'm not sure exactly why this happens with the plugin, nobody else I've talked to seems to have this issue but I can't really provide a copy of my repository for debugging.
